opm/simulators/wells/GroupEconomicLimitsChecker.hpp

```diff
@@ -74,6 +80,11 @@ namespace Opm
             {1, BlackoilPhases::Vapour},
             {2, BlackoilPhases::Aqua}};
         std::map<BlackoilPhases::PhaseIndex, int> phase_idx_reverse_map_;
+        const std::string date_string_;
+        const UnitSystem& unit_system_;
```

Comment:
Note that this declaration order&ndash;after phase_idx_reverse_map_&ndash;is at odds with the apparent [initialisation order](https://github.com/OPM/opm-simulators/blob/c3d006e91c14c9878d352982f7d02cf0d4bc406b/opm/simulators/wells/GroupEconomicLimitsChecker.cpp#L60-L61) in `GroupEconomicsLimitsChecker.cpp`&ndash;between `deferred_logger_` and `well_state_`.  That, in turn, generates some additional warnings that I'd prefer we avoid.  I think the simplest approach would be to just move the declarations of `date_string_` and `unit_system_` up to the location implied by the textual initialisation order in the `.cpp` file.